### PR TITLE
New theme Aletheia

### PR DESCRIPTION
**Pull request by silkfire**
This is a my long time go-to theme for Visual Studio, inspired by the [Son of Obsidian theme](https://studiostyl.es/schemes/son-of-obsidian).

image

### CODE REVIEW

File: VSColorThemes/Themes/Aletheia.xml [hunk 1]
```diff
@@ -0,0 +1,11366 @@
...
+      <Color Name="LiveMetricsFailedRequestFill">
+        <Background Type="CT_RAW" Source="FFFCECEE" />
+      </Color>
+      <Color Name="SquigglyOrange">
+        <Background Type="CT_RAW" Source="00FF7F29" />
+      </Color>
+    </Category>
+    <Category Name="Autos" GUID="{a7ee6bee-d0aa-4b2f-ad9d-748276a725f6}">
+      <Color Name="ChangedText">
+        <Background Type="CT_INVALID" Source="00000000" />
```
Comment (Newrad0603) [inline VSColorThemes/Themes/Aletheia.xml]: Please remove all instances of CT_INVALID color entries. They are non-functional, but bloat the private registry.

File: VSColorThemes/Themes/Aletheia.xml [hunk 2]
```diff
@@ -0,0 +1,11366 @@
...
+        <Foreground Type="CT_RAW" Source="FFF7A1A1" />
+      </Color>
+      <Color Name="Plain Text">
```
Comment (Newrad0603) [inline VSColorThemes/Themes/Aletheia.xml]: Once the CT_INVALID entries are removed, you'll end up with empty colors, so make sure to remove those as well. Once the empty colors are removed, you may have empty categories which should also be removed.

**Comment by silkfire** [comment]
Why did it take 6 months for someone to review this PR?

**Comment by Newrad0603** [comment]
@silkfire I apologize for the lack of activity, the previous owner of the repo left Microsoft, and this fell through the gaps in terms of ownership transfer.

File: VSColorThemes/Themes/Aletheia.xml (hunk 1 above)
Comment (silkfire) [reply VSColorThemes/Themes/Aletheia.xml]: Removed all `CT_INVALID` entries...

File: VSColorThemes/Themes/Aletheia.xml (hunk 2 above)
Comment (silkfire) [reply VSColorThemes/Themes/Aletheia.xml]: ...and any resulting empty nodes.

**Comment by silkfire** [comment]
@Newrad0603 See my updates :)

**Comment by silkfire** [comment]
@Newrad0603 Thanks Andrew. When is the next planned release / publish to VS Marketplace?

**Pull request merged**